MbDotNet/Models/Predicates/InjectPredicate.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Newtonsoft.Json;
+
+namespace MbDotNet.Models.Predicates
+{
+    public class InjectPredicate : PredicateBase
```

Comment:
It doesn't make a lot of sense for this to extend `PredicateBase` since none of the other properties on there are relevant for injection, but there isn't really a way around it since the predicates are stored in a collection of `PredicateBase` on the stub. We'll leave this for now and I'll consider a larger refactor to fix it later if it is worth it.